tagsets/query.php

```diff
@@ -861,7 +861,8 @@ function query__apply_permanent_queries() {
         $query_assigned=array();
         foreach ($ppart as $p) {
             $num_p++;
-            foreach ($pqu as $q) {
+	    foreach ($pqu as $q) {
```

Comment:
This probably need indenting, so it is not at the same level as outer foreach.

Reply:
Oh yes, this get messed up with tabs instead of space. Fixed now.